src/main/java/org/assertj/core/api/AbstractByteArrayAssert.java

```diff
@@ -220,6 +221,32 @@ public SELF contains(byte... values) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array contains the given values, in any order.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertion will pass
+   * assertThat(new byte[] { 1, 2, 3 }).contains(new Byte[] { 1,  2});
+   * assertThat(new byte[] { 1, 2, 3 }).contains(new Byte[] { 3,  1});
+   * assertThat(new byte[] { 1, 2, 3 }).contains(new Byte[] { 1,  3,  2});
+   *
+   * // assertion will fail
+   * assertThat(new byte[] { 1, 2, 3 }).contains(new Byte[] { 1,  4});
+   * assertThat(new byte[] { 1, 2, 3 }).contains(new Byte[] { 4,  7});</code></pre>
+   *
+   * @param values the given values.
+   * @return {@code this} assertion object.
+   * @throws NullPointerException     if the given argument is {@code null}.
+   * @throws IllegalArgumentException if the given argument is an empty array.
+   * @throws AssertionError           if the actual array is {@code null}.
+   * @throws AssertionError           if the actual array does not contain the given values.
```

Comment:
Missing `@since`:
```suggestion
   * @throws AssertionError           if the actual array does not contain the given values.
   * @since 3.19.0
```